import-service/services/import-file-parser/importFileParser.api.ts

```diff
@@ -1,17 +1,53 @@
 import { S3Handler } from 'aws-lambda';
+import { SQS } from 'aws-sdk';
 import { ApiGatewayError } from '../../../utils/ApiGatewayError';
 import { importFileParserService } from './importFileParser.service';
 
-export const importFileParser: S3Handler = async (event) => {
+export const importFileParser: S3Handler = async (event, context, callback) => {
+  console.log('🚀 ~ file: importFileParser.api.ts ~ line 7 ~ event', event);
   try {
-    console.log({ event });
     for (const record of event.Records) {
-      console.log('Record: ', record);
+      console.log(
+        '🚀 ~ file: importFileParser.api.ts ~ line 10 ~ record',
+        record
+      );
       const {
         bucket: { name: bucketName },
         object: { key, eTag },
       } = record.s3;
-      await importFileParserService({ bucketName, key });
+      const parsedCsv = await importFileParserService({ bucketName, key });
+
+      const sqs = new SQS();
+      await Promise.all(
+        parsedCsv.map(async (csvRecord) => {
+          console.log(
+            '🚀 ~ file: importFileParser.api.ts ~ line 23 ~ csvRecord',
+            csvRecord
+          );
+          console.log(
+            `start of sending message ${JSON.stringify(csvRecord)} to SQS`
+          );
+          const sqsUrl = process.env.SQS_URL;
+          await sqs
```

Comment:
The same as for previous handler - no need to use callback, since you already use try-catch with await sqs.sendMessage...

Reply:
Thanks, remove callback